### PR TITLE
added zdf.de

### PR DESCRIPTION
**Pull request by LethalSnake1337**

### CODE REVIEW

**Comment by MaxDall** [comment]
Hey @LethalSnake1337 thanks for adding this :) looks good 👍 We recently added a new attribute to our unit tests today. Could you merge the latest master branch into your branch and run `python -m scripts.generate_parser_test_files -p ZDF` once again.

**Comment by LethalSnake1337** [comment]
> Hey @LethalSnake1337 thanks for adding this :) looks good 👍 We recently added a new attribute to our unit tests today. Could you merge the latest master branch into your branch and run `python -m scripts.generate_parser_test_files -p ZDF` once again.

Hey @MaxDall done 👍.

**Comment by LethalSnake1337** [comment]
I added a summary selector and generated a new unit test

File: src/fundus/publishers/de/zdf.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class ZDFParser(ParserProxy):
+    class V1(BaseParser):
+        _div_selector = CSSSelector("div.r1nj4qn5")
+        _summary_selector = CSSSelector("p.ikh9v7p.c1bdz7f4")
```
Comment (MaxDall) [inline src/fundus/publishers/de/zdf.py:17]: The articles seem to include subheadlines as well.
https://www.zdf.de/nachrichten/politik/ausland/trump-johnson-ukraine-hilfen-100.html
https://www.zdf.de/nachrichten/panorama/busunfall-klassenfahrt-jugendliche-verletzte-marburg-100.html

Would be good to add a `_subheadline_selector` as well.

**Review by MaxDall — CHANGES_REQUESTED** [review]

File: src/fundus/publishers/de/zdf.py (hunk 1 above)
Comment (LethalSnake1337) [reply src/fundus/publishers/de/zdf.py:17]: Added the `_subheadline_selector` as well. Let me know if anything is missing 👍

**Review by MaxDall — APPROVED** [review]

**Pull request merged**